core/templates/dev/head/pages/exploration_player/ProgressNavDirective.js

```diff
@@ -100,20 +106,20 @@ oppia.directive('progressNav', [
 
             return (interactionHasNavSubmitButton && (
               interactionIsInline ||
-              !ExplorationPlayerService.canWindowShowTwoCards()
+              !$scope.canWindowShowTwoCards()
             ));
           };
 
           $scope.shouldContinueButtonBeShown = function() {
             if ($scope.conceptCardIsBeingShown) {
               return true;
             }
-            var lastPair = $scope.activeCard.inputResponsePairs[
-              $scope.activeCard.inputResponsePairs.length - 1];
+            var lastPair = $scope.activeCard.getInputResponsePairs()[
```

Comment:
$scope.activeCard.getLastInputResponsePair()

Reply:
Done, used the getLastOppiaResponse()..